transaction/views.py

```diff
@@ -86,16 +94,42 @@ def __getUserCartId(user_id):
     return new_cart_id
 
 def __updateStatus(cart_id):
-        with connection.cursor() as c:
-            c.execute('''
-                UPDATE cart
-                SET is_completed = true
-                WHERE id = %s
-            ''', [cart_id])
+    with connection.cursor() as c:
+        c.execute('''
+            UPDATE cart
+            SET is_completed = true
+            WHERE id = %s
+        ''', [cart_id])
 
 def __deleteItem(cart_id, cart_item_id):
-        with connection.cursor() as c:
-            c.execute('''DELETE FROM cart_item 
-                         WHERE cart_id = %s AND 
-                               object_id = %s
-            ''', [cart_id, cart_item_id])
+    with connection.cursor() as c:
+        c.execute('''DELETE FROM cart_item 
+                     WHERE cart_id = %s AND 
+                           object_id = %s
+        ''', [cart_id, cart_item_id])
+
+def __addItem(user_id, multimedia_id, multimedia_type):
+    content_type = ContentType.objects.get(model=multimedia_type)
+    cart_id = __getUserCartId(user_id)
+    with connection.cursor() as c:
+        c.execute('''INSERT INTO cart_item(cart_id, object_id, content_type_id) 
```

Comment:
Does this method returns the id of the newly created cart item? If yes, it would be good to return this value.